Formula/influxdb.rb

```diff
@@ -131,10 +127,10 @@ def plist
                              "--http-bind-address=:#{influxd_port}",
                              "--log-level=error"
     end
-    sleep 20
+    sleep 30
```

Comment:
Bumped because CI failed once with a "connection refused", showing the DB didn't come up quickly enough. It'd be nice to instead have a loop that tries until success or some max attempt count, so the test can move on ASAP if the DB doesn't take a full 30 seconds to boot up. I can try implementing that if there's interest.

Reply:
I think this is fine for now. If it starts taking longer than that we can revisit this.